docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EE_Application_Deployment_Configuration.adoc

```diff
@@ -39,6 +39,71 @@ The optional `meta-inf` attribute, which defaults to `true`, indicates
 if the Module's `META-INF` path should be available to the deployment's
 class loader.
 
+[[global-directories]]
+== Global Directories
```

Comment:
I think the title should be Global Directory, since now there can be only one.

Reply:
This was changed to Global directory